src/licensedcode/data/rules/lua.RULE

```diff
@@ -0,0 +1,5 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
This looks now as exactly an standard MIT license.
Try to run the detection on that text on the develop brancj

Reply:
I ran license detection on it and it shows only 17% relevance to the MIT Modern license among a few others (like NewsLetr, Nillsson, etc) so made it a rule for the MIT modern license. However, if you think that I should remove the rule and YAML files for Lua, I'll do it right away. Thanks a lot for the clarification!